RedHat-UX/red-hat-design-system
# fix(avatar): default image file sizes

## What I did

1. Replaced default base 64 PNGs with SVGs in `rh-avatar`, resolving #1153 


## Testing Instructions

1. Check [Sample Component in DP](https://deploy-preview-1154--red-hat-design-system.netlify.app/elements/avatar/#sample-component).
2. Verify that [file size](https://github.com/RedHat-UX/red-hat-design-system/pull/1154#issuecomment-1644263472) is significantly smaller.

## Notes to Reviewers
